packages/data/src/api/tokenById/tokenById.test.ts

```diff
@@ -0,0 +1,98 @@
+import { tokenById, checkParams } from "./tokenById";
+import { TokenByIdResults } from "./tokenById.types";
+
+import {
+  errorContractAddress,
+  errorToken,
+  tokenByIdMock,
+} from "./tokenById.mock";
```

Comment:
Interesting linter didn't fix quotes, it should lint tests as well will double check that scripting.  I don't have huge opinions either way on it, but went with [single quotes](https://github.com/Mintbase/mintbase-js/blob/modular-refactor/.eslintrc.json#L42 )